opnode/eth/heads.go

```diff
@@ -0,0 +1,48 @@
+package eth
+
+import (
+	"context"
+
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/event"
+)
+
+type HeadSignal struct {
+	Parent BlockID
+	Self   BlockID
+}
+
+// HeadSignalFn is called to inform a chain with a new head
+type HeadSignalFn func(sig HeadSignal)
+
+// WatchHeadChanges wraps a new-head subscription from ChainReader to feed the given Tracker
```

Comment:
What is `ChainReader`? Outdated comment and it's now `NewHeadSource`?
Same with `Tracker` - `HeadSignalFn`?

Reply:
`ChainReader` is a super common ethereum interface, in the very root package: https://github.com/ethereum/go-ethereum/blob/045e90c8971cddbabeac0abd54240cf02bc1d94d/interfaces.go#L54
Will update with package name to clarify where it's coming from.